segmentstore/server/src/main/java/io/pravega/segmentstore/server/containers/StreamSegmentContainer.java

```diff
@@ -229,7 +229,12 @@ private MetadataStore createMetadataStore() {
             ChunkedSegmentStorage chunkedStorage = (ChunkedSegmentStorage) this.storage;
             val snapshotInfoStore = getStorageSnapshotInfoStore();
             // Bootstrap
-            return chunkedStorage.bootstrap(snapshotInfoStore);
+            // Note: Temp change to be removed after next PR (part 2 of 3)
```

Comment:
Please put a TODO here and link to a GitHub issue.

Or, alternatively, create a feature branch to collect your PRs in it.

Reply:
The second PR depended on Raul's change.
Now that that PR is in, I can combine part 1 and 2 into a single PR.